src/components/Select/Select/Select.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { useSelect } from 'downshift';
+import { IoMdArrowDropup, IoMdArrowDropdown } from 'react-icons/io';
+import { bem } from '../../../utils/bem/bem';
+import { FieldWrapper } from '../../FieldWrapper';
+import { List } from '../../List';
+import { BasicSelectProps } from '../SelectBase';
+import { SuggestionsList } from '../SuggestionsList';
+import styles from './Select.scss';
+
+const { block, elem } = bem('Select', styles);
+
+interface Props<S> extends Omit<BasicSelectProps<S>, 'isProminent'> {
+    /** The suggestions that is currently selected */
+    selectedSuggestion: S;
+}
+
+export function Select<S>(props: Props<S>) {
+    const {
+        suggestions,
+        selectedSuggestion,
+        suggestionToString,
+        suggestionItemRenderer,
+        onSelectionAdd,
+        onBlur,
+        onFocus,
+        rootRef,
+        listRef,
+        ...rest
+    } = props;
```

Comment:
There's a bug of Select component being closed once it's opened and the browser tab is switched. It can be fixed with `useBrowserTabVisibilityChange`. You can take ComboboxMulti component as an example.

Reply:
Our custom hook doesn't capture the event. Also found that downshift is inconsistent in its state update in this case, meaning sometimes it collapses, and sometimes not. After offline discussions with both Ihor and Carlo, created an issue in their repo: https://github.com/downshift-js/downshift/issues/1157